tests/go/GoErrCheckBearTest.py

```diff
@@ -0,0 +1,59 @@
+from bears.go.GoErrCheckBear import GoErrCheckBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+good_file = """package main
+import "fmt"
+func main() {
+    fmt.Println("Hello, Arch!")
+}
+""".splitlines(keepends=True)
+
+
+bad_file = """package main
+import "os"
+func main() {
+f, _ := os.Open("foo")
+f.Write([]byte("Hello, world."))
+f.Close()
+}
+""".splitlines(keepends=True)
+
+GoErrCheckBearTest = verify_local_bear(GoErrCheckBear,
+                                       valid_files=(good_file,),
+                                       invalid_files=(bad_file,),
+                                       tempfile_kwargs={"suffix": ".go"})
+
+GoErrCheckBearWithSetting = verify_local_bear(GoErrCheckBear,
```

Comment:
These class names don't end with "Test"
Last I remember, pytest finds classes ending with "Test"
Are you sure these tests are even being run ?